### DISCUSSION TITLE
CAP V1.2 API (Active Endpoint Interruption)

### DISCUSSION THREAD

**Discussion by grfritz** [weather-gov/api#377] [Q&A]
In case anyone noticed, there was a one hour interruption in active endpoint delivery yesterday afternoon (12/13/19) starting at 21:08:12 GMT.  New alert postings halted at that time and resumed at 22:11:12 GMT with 54 delayed alerts (which accumulated during the outage) posted.  A check of the diagnostic logs revealed product was still flowing from HCE but was not being ingested or cataloged by the API.  Since that time normal product flow has resumed.  This information has also been conveyed to NCEP SDB developers.

**Comment by StephenClouse** [comment]
Operational issues such as data outages should be reported to the TOC. We cannot support operational issues via this forum.
